packages/wells/src/__tests__/cogniteClient.int.spec.ts

```diff
@@ -2,8 +2,16 @@
 
 import CogniteClient from '../cogniteClient';
 import { setupLoggedInClient } from './testUtils';
+import { isNullOrUndefined } from 'util';
```

Comment:
`isNullOrUndefined` is deprecated
```
process.env.COGNITE_WELLS_PROJECT && process.env.COGNITE_WELLS_CREDENTIALS
```
should be enough

Reply:
changed :+1: